src/common/accelcmn.cpp

```diff
@@ -156,6 +156,96 @@ static int IsNumberedAccelKey(const wxString& str,
     return prefixCode + num - first;
 }
 
+// static
+bool wxAcceleratorEntry::ValidateKey(int flags, int keycode)
+{
+    wxString keyname;
+    if ( keycode )
+    {
+        for ( size_t n = 0; n < WXSIZEOF(wxKeyNames); n++ )
+        {
+            const wxKeyName& kn = wxKeyNames[n];
+            if ( kn.code == keycode )
+            {
+                keyname = kn.name;
+                break;
+            }
+        }
+
+        if ( keyname.IsEmpty() && keycode >= WXK_SPECIAL1 && keycode <= WXK_SPECIAL20 )
+            keyname << wxS("SPECIAL")  << keycode - WXK_SPECIAL1 + 1;
+    }
+
+    bool valid = true;
+    switch ( keycode )
+    {
+        /*
+         The following keycodes must have modifier keys to be valid on GTK
+         */
+        case WXK_UP:
+        case WXK_DOWN:
+        case WXK_LEFT:
+        case WXK_RIGHT:
+        case WXK_NUMPAD_UP:
+        case WXK_NUMPAD_DOWN:
+        case WXK_NUMPAD_LEFT:
+        case WXK_NUMPAD_RIGHT:
+            if ( !flags )
+            {
+                valid = false;
+                wxLogDebug( wxString::Format( wxT("Compatibility issue: %s key must have modifiers to be an accelerator key on GTK"),
```

Comment:
```suggestion
                wxLogDebug("Compatibility issue: %s key must have modifiers to be an accelerator key on GTK",
```
i.e. there is no need for `wxString::Format()` as `wxLogXXX` functions are already (pseudo-)vararg and `wxT()` shouldn't be used in the new code (it can be replaced with `wxS` if performance is a concern, but can also be omitted entirely).